config/routes.rb

```diff
@@ -6,6 +6,8 @@
     namespace :v1 do
       resources :orders do
          get 'cart', on: :collection, to: 'orders#cart'
```

Comment:
You shouldn't need the `to: 'orders#cart'`, rails will figure it out cuz its the same name